src/backend/joanie/tests/core/test_api_order.py

```diff
@@ -1187,16 +1203,15 @@ def test_api_order_create_once(self):
 
         self.assertEqual(response.status_code, 201)
 
-    def test_api_order_create_payment_requires_billing_address(self):
+    def test_api_order_create_payment_does_not_requires_billing_address(self):
```

Comment:
Furthermore, it misses a test to check that is not possible to submit an order without billing address if the related product is fee.